### DISCUSSION TITLE
outfit rating checkkkkk (1-10)

### DISCUSSION THREAD

**Discussion by SakuraKitsunee** [Prodigy-Hacking/ProdigyMathGameHacking#937] [(A) Anything]
rate my my outfit (1-10)

![image](https://user-images.githubusercontent.com/75956379/118002985-037e2200-b2fd-11eb-9901-cf377340a071.png)

**Comment by ghost** [comment]
8. masterr of trials circlet and miras robes plus trialsmasters boots with chill and char+astrall relic is a 10
